Package/Android/build.gradle

```diff
@@ -33,6 +33,7 @@ buildscript {
             jcenter()
         }
         dependencies {
+// TODO move to 4.1.0?
```

Comment:
For me locally, updating this gradle version didn't seem needed. @ryantrem would you like to keep it at 3.4.1 or move it to 4.1.0? I personally am not sure what impacts the change has on npm package consumers

Reply:
Leave this one as is I think.